police organizations (Kiselev 821701)

Nikkis236 opened the pull request:
The translator itself will be committed later

Nikkis236 committed:
Added police organizations (generated with translator)

The translator itself will be committed later

Nikkis236 committed:
Add concept and translator for police offices

Nikkis236 commented:
Добавлены все полицейские учреждения (200+) и транслятор для их получения

Vikort reviewed (commented):
Первая часть принята
В названии пр достаточно темы, остальное можешь добавить в описании, как минимум фамилию
Пр смержим после принятия всех частей

Nikkis236 committed:
Added 200 factories

Добавил 200 понятий заводов от Кандыбы Е.С. и Зайцева Н.А. 821702. У них проблемы с подключением к github.

Nikkis236 commented:
У Зайцева Н.А. и Кандыбы Е.С. проблемы с подключением к github. Поэтому по их просьбе залил сюда их 200 понятий из предметной области "заводы"

Nikkis236 committed:
Added interface

Nikkis236 commented:
Добавлен интерфейс для вызова транслятора

Vikort commented:
Понятия Зайцева Н.А. и Кандыбы Е.С. не собираются

Vikort commented on interface/sc-web-extensions/police_translator/police_translator/kb/lib_component_ui_menu_police_translator.scs:
```diff
@@ -0,0 +1,10 @@
+	(* <- lang_en;; *);
+
+<- library_of_platform_independent_reusable_components;
+<- library_of_atomic_reusable_components;;
+<- library_of_atomic_reusable_components;;
```
```suggestion
<- library_of_atomic_reusable_components;;
```

Vikort commented on interface/sc-web-extensions/police_translator/police_translator/kb/format_police_translator.scs:
```diff
@@ -0,0 +1,27 @@
+	[format of UI component police_translator]
+	(*
+	<- lang_en;;
+	*);m
```
```suggestion
	*);
```

Vikort commented on kb/organization/police/minsk/obshchestvennyy_punkt_okhrany_pravoporyadka_n 154.scs:
```diff
@@ -0,0 +1,6 @@
+obshchestvennyy_punkt_okhrany_pravoporyadka_n 154 <- concept_police;
```
```suggestion
obshchestvennyy_punkt_okhrany_pravoporyadka_n_154 <- concept_police;
```

Vikort commented on kb/organization/police/mogilev/udkse_pa_magіlëўskay_voblastsі.scs:
```diff
@@ -0,0 +1,6 @@
+udkse_pa_magіlëўskay_voblastsі <- concept_police;
```
```suggestion
udkse_pa_magileuskay_voblastsi <- concept_police;
```

Vikort commented on problem-solver/py/services/PoliceTranslatorModule/specification/lib_component_ui_menu_police_translator.scs:
```diff
@@ -0,0 +1,10 @@
+	(* <- lang_en;; *);
+
+<- library_of_platform_independent_reusable_components;
+<- library_of_atomic_reusable_components;;
+<- library_of_atomic_reusable_components;;
```
```suggestion
<- library_of_atomic_reusable_components;;
```

Vikort reviewed (commented):
Все принято
Смержу после исправлений замечаний

Nikkis236 committed:
Fixed remarks

Nikkis236 commented:
Исправил замечания
